### PR TITLE
v3.7.5

### PR DESCRIPTION
**Pull request by superboyiii**
v3.7.5 hotfix

### CODE REVIEW

**Review by AnnaShaleva — APPROVED** [review]

**Review by Jim8y — APPROVED** [review]

**Comment by vncoelho** [comment]
Merge https://github.com/neo-project/neo/pull/3320 before, for tests locally to work

**Comment by vncoelho** [comment]
https://github.com/neo-project/neo/pull/3306/files

I mean this one

File: src/Neo.CLI/config.mainnet.json [hunk 1]
```diff
@@ -37,7 +37,8 @@
     "Hardforks": {
       "HF_Aspidochelone": 1730000,
       "HF_Basilisk": 4120000,
-      "HF_Cockatrice": 5450000
+      "HF_Cockatrice": 5450000,
+      "HF_Domovoi": 5570000
```
Comment (AnnaShaleva) [inline src/Neo.CLI/config.mainnet.json:41]: Could you please set the Domovoi height to 5800000 for NeoFS mainnet config? See the https://github.com/nspcc-dev/neo-go/pull/3487#discussion_r1635193295.

File: .github/workflows/main.yml
Comment (cschuchardt88) [inline .github/workflows/main.yml:1]: Your going to have to change this line 104
```yml
if: github.ref == 'refs/heads/master' && startsWith(github.repository, 'neo-project/')
```

**Review by cschuchardt88 — CHANGES_REQUESTED** [review]

File: .github/workflows/main.yml
Comment (shargon) [reply .github/workflows/main.yml:1]: I think @cschuchardt88 is right

Comment (superboyiii) [reply .github/workflows/main.yml:1]: Done.

File: src/Neo.CLI/config.mainnet.json (hunk 1 above)
Comment (superboyiii) [reply src/Neo.CLI/config.mainnet.json:41]: Sure

**Comment by superboyiii** [comment]
@AnnaShaleva @cschuchardt88 Could you review again?

**Comment by superboyiii** [comment]
> https://github.com/neo-project/neo/pull/3306/files
> 
> I mean this one

Not criticial for this hotfix. We can put it into v3.8.0

**Review by cschuchardt88 — APPROVED** [review]

File: src/Neo.CLI/config.fs.testnet.json [hunk 2]
```diff
@@ -36,6 +36,12 @@
     "MaxTraceableBlocks": 2102400,
     "InitialGasDistribution": 5200000000000000,
     "ValidatorsCount": 7,
+    "Hardforks": {
+      "HF_Aspidochelone": 210000,
+      "HF_Basilisk": 2680000,
+      "HF_Cockatrice": 3967000,
+      "HF_Domovoi": 4144000
+    },
```
Comment (AnnaShaleva) [inline src/Neo.CLI/config.fs.testnet.json:44]: For NeoFS testnet this section is not needed, we have all hardforks enabled starting from the genesis. Just remove it.

**Review by AnnaShaleva — COMMENTED** [review]
Otherwise LGTM.

File: src/Neo.CLI/config.fs.testnet.json (hunk 2 above)
Comment (superboyiii) [reply src/Neo.CLI/config.fs.testnet.json]: Removed

**Review by AnnaShaleva — APPROVED** [review]

**Review by AnnaShaleva — APPROVED** [review]

**Review by shargon — COMMENTED** [review]
LgTM

**Pull request merged**